mission-control/docs/installation/self-hosted/database.md

```
@@ -63,3 +63,14 @@ db:
     name: mission-control-postgres
     key: DB_URL
 ```
+
+### Google Cloud SQL
+
+When you use Google Cloud SQL for PostgreSQL, grant the Mission Control database user the permissions required by startup migrations before you install or upgrade Mission Control:
+
+```sql
+GRANT cloudsqlsuperuser TO your_username;
+ALTER ROLE your_username CREATEROLE;
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)